app/jobs/calendar_importer/parsers/ld_json.rb

```diff
@@ -55,83 +55,19 @@ class EventConsumer
         'WebSite' => :consume_website
       }.freeze
 
-      def consume_business_event(data)
-        consume_event data
-      end
-
-      def consume_childrens_event(data)
-        consume_event data
-      end
-
-      def consume_comedy_event(data)
-        consume_event data
-      end
-
-      def consume_dance_event(data)
-        consume_event data
-      end
-
-      def consume_delivery_event(data)
-        consume_event data
-      end
-
-      def consume_education_event(data)
-        consume_event data
-      end
-
-      def consume_exhibition_event(data)
-        consume_event data
-      end
-
-      def consume_food_event(data)
-        consume_event data
-      end
-
-      def consume_literary_event(data)
-        consume_event data
-      end
+      event_types = %w[business childrens course_instance comedy dance delivery education event_series exhibition festival food hackathon literary music publication sale screening social sports theatre visual_arts]
 
-      def consume_music_event(data)
-        consume_event data
-      end
-
-      def consume_publication_event(data)
-        consume_event data
-      end
-
-      def consume_sale_event(data)
-        consume_event data
-      end
-
-      def consume_screening_event(data)
-        consume_event data
-      end
-
-      def consume_social_event(data)
-        consume_event data
-      end
-
-      def consume_sports_event(data)
-        consume_event data
-      end
-
-      def consume_theater_event(data)
-        consume_event data
-      end
-
-      def consume_visual_arts_event(data)
-        consume_event data
+      event_types.each do |type|
+        define_method(:"consume_#{type}_event") do |data|
+          consume_event data
+        end
       end
 
-      def consume_course_instance(data); end
```

Comment:
Very unclear why these are not implemented, assume this was an oversight but making a comment so someone can double check